README.md

```diff
@@ -2,20 +2,28 @@
 *Professor X: For someone who hates mutants... you certainly keep some strange company.*  
 *William Stryker: Oh, they serve their purpose... as long as they can be controlled.*
 
-## Getting started
-Stryker.NET offers you mutation testing for your .NET Core projects. It allows you to test your tests by temporarily inserting bugs.
+## Introduction
+
+For an introduction to mutation testing and Stryker's features, see [stryker-mutator.io](https://stryker-mutator.io/).
 
 ## Note
-This project is still in its early days and is not yet available on NuGet
+This project is still in its early days and is not yet available on NuGet. In the meantime, start by [mutation testing your JavaScript](https://stryker-mutator.github.io).
+
+## Getting started
+Stryker.NET offers you mutation testing for your .NET Core projects. It allows you to test your tests by temporarily inserting bugs.
 
 ## Contributing
 Want to help develop Stryker.NET? Check out our [contribution guide](/CONTRIBUTING.md).
-In the meantime, start by [mutation testing your JavaScript](https://stryker-mutator.github.io)
 
 ## Compatibility
 Only compatible with .NET Core version 1.1+
 
+## Usage
```

Comment:
The info in this readme is not grouped together. I would expect the usage to be described in/near `Getting Started`